### PR TITLE
solution hw-HB-01

### PR DESCRIPTION
**Pull request by RomanHreits**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+language: java
+jdk:
+  - openjdk11
+script:
+  - mvn clean install
+cache:
+  directories:
+    - $HOME/.m2
```
Comment (andriylagun) [inline .travis.yml]: Add empty line

File: src/main/resources/log4j2.xml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    </Appenders>
+    <Loggers>
+        <Logger name="hibernate-cinema" level="debug" additivity="true">
+            <appender-ref ref="fileLogger" level="debug" />
+        </Logger>
+        <Root level="debug" additivity="false">
+            <appender-ref ref="console" />
+        </Root>
+    </Loggers>
+</Configuration>
```
Comment (andriylagun) [inline src/main/resources/log4j2.xml]: same

File: src/main/resources/hibernate.cfg.xml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        <property name="connection.username">root</property>
+        <property name="connection.password">romanhreits@199122</property>
+        <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>
+        <property name="show_sql">true</property>
+        <property name="hbm2ddl.auto">create-drop</property>
+
+        <mapping class="com.cinema.model.Movie"/>
+    </session-factory>
+
+</hibernate-configuration>
```
Comment (andriylagun) [inline src/main/resources/hibernate.cfg.xml]: emptyline

**Review by andriylagun — APPROVED** [review]
Well done. Minor comments

File: src/main/java/com/cinema/exceptions/DataProcessingException.java [hunk 4]
```diff
@@ -0,0 +1,13 @@
+package com.cinema.exceptions;
+
+public class DataProcessingException extends RuntimeException {
+    private String message;
```
Comment (KseniiaMakarova) [inline src/main/java/com/cinema/exceptions/DataProcessingException.java]: do we need this field?

File: src/main/java/com/cinema/util/HibernateUtil.java [hunk 5]
```diff
@@ -0,0 +1,23 @@
+package com.cinema.util;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {
+    public static SessionFactory factory = initSessionFactory();
```
Comment (KseniiaMakarova) [inline src/main/java/com/cinema/util/HibernateUtil.java]:
```suggestion
    private static SessionFactory factory = initSessionFactory();
```

File: src/main/resources/hibernate.cfg.xml [hunk 6]
```diff
@@ -0,0 +1,20 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE hibernate-configuration PUBLIC
+        "-//Hibernate/Hibernate Configuration DTD 5.3//EN"
+        "http://hibernate.sourceforge.net/hibernate-configuration-5.3.dtd">
+
+<hibernate-configuration>
+
+    <session-factory>
+        <property name="hibernate.dialect">org.hibernate.dialect.MySQL5Dialect</property>
+        <property name="connection.url">jdbc:mysql://localhost/cinema?serverTimezone=UTC</property>
```
Comment (KseniiaMakarova) [inline src/main/resources/hibernate.cfg.xml]: does it use 3306 by default?

File: pom.xml [hunk 7]
```diff
@@ -40,6 +67,14 @@
                     <linkXRef>false</linkXRef>
                 </configuration>
             </plugin>
+            <plugin>
```
Comment (KseniiaMakarova) [inline pom.xml]: let's add `<pluginManagement>` block

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/resources/hibernate.cfg.xml (hunk 6 above)
Comment (RomanHreits) [reply src/main/resources/hibernate.cfg.xml]: і так все працювало, але я добавив

File: src/main/java/com/cinema/util/HibernateUtil.java (hunk 5 above)
Comment (RomanHreits) [reply src/main/java/com/cinema/util/HibernateUtil.java]: done

File: src/main/java/com/cinema/exceptions/DataProcessingException.java (hunk 4 above)
Comment (RomanHreits) [reply src/main/java/com/cinema/exceptions/DataProcessingException.java]: done

File: pom.xml (hunk 7 above)
Comment (RomanHreits) [reply pom.xml]: done

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**